### ISSUE TITLE
Notification wall

### ISSUE THREAD

**Issue by namangupta01** [publiclab/plots2#5745]
We also can have a place where a user can see all the notification just like facebook, Quora, twitter etc.

Let's discuss @gautamig54 and @jywarren !

**Comment by gautamig54** [comment]
Yes @namangupta01. I was talking about this only. This will give a better way for users to track their feed and activities. There are various gems which we can use to implement this. I have mentioned them in #5400 
@jywarren What do think about this?

**Comment by gautamig54** [comment]
I will come up with a mock up design of the notification wall in a day or two.

**Comment by namangupta01** [comment]
Cool!

**Comment by namangupta01** [comment]
Hey @gautamig54 Are you working on it? Do you need some help here?
